web/app/controllers/bloggs_controller.rb

```diff
@@ -12,17 +12,19 @@ def show
 
   def new
     @article = Blogg.new
+    authorize! :new, @article
```

Comment:
Ok, that's essentially how I understood it, thanks for making sure I'm on the same page with things ;)